docker/oracle/setup.sql

```diff
@@ -0,0 +1,20 @@
+alter profile default limit password_life_time unlimited;
+
+CREATE USER nucore_open_development
+    IDENTIFIED BY password
+    DEFAULT TABLESPACE users
```

Comment:
This file will change somewhat in NU. We'll want to change the usernames, and NU should use `bc_nucore` as the tablespace.